packages/core/src/definitions/styles.ts

```diff
@@ -56,6 +56,13 @@ export const builtInStyles: VariableDefinitions = {
     defaultValue: 'center',
     displayName: 'Horizontal alignment',
   },
+  cfVisibility: {
+    displayName: 'Visibility toggle',
+    type: 'Boolean',
+    group: 'style',
+    defaultValue: false,
+    description: 'To enable visibility toggle in patterns',
```

Comment:
Is it only going to apply to the patterns or to all registered components? 🤔

Reply:
Thanks for pointing that out